src/Altinn.Apps/AppTemplates/AspNet/App/logic/Calculation/CalculationHandler.cs

```diff
@@ -22,7 +23,7 @@ public CalculationHandler()
         /// }
         /// </example>
         /// <param name="instance">The data</param>
-        public bool Calculate(object instance)
+        public async Task<bool> Calculate(object instance)
         {
             return false;
```

Comment:
This will now give compile warnings as the method does not use await. In these cases we can use return Task.FromResult(false); or similar.